src/com/khadri/hibernate/college/entity/College.java

```diff
@@ -0,0 +1,28 @@
+package com.khadri.hibernate.college.entity;
```

Comment:
change package from com.khadri.hibernate.college.entity to com.khadri.hibernate.entity